mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +27,55 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enlarge Image
+   */
+  enlargeImage?: boolean;
+
+  /**
+   * Enlarge Image Styles
+   */
+  enlargeClassName?: string;
 }
 
 /**
  * Сomponent for displaying images
  */
 export const Image = (props: ImageProps) => {
   const className = clsx(styles.image, props.className);
+  const enlargeClassName = clsx(styles.image, props.enlargeClassName);
 
   return (
-    <img
-      src={props.src}
-      alt={props.alt}
-      className={className}
-      data-cy={props.dataCy}
-    />
+    props.enlargeImage
+      ? (
+        <Modal
+          trigger={
+            <img
+              src={props.src}
+              alt={props.alt}
+              className={className}
+              data-cy={props.dataCy}
+            />
+          }
```

Comment:
let's move img... above return and reuse one element instead of write img 3 times)